source/includes/fundamentals/code-snippets/CollationCollectionExample.java

```diff
@@ -190,12 +193,13 @@ private static void operationCollation(MongoCollection collection) {
 
     public static void main(String[] args) {
 
-        String uri = "<MongoDB connection URI>";
+        String uri = "mongodb://localhost:27017";
```

Comment:
Did you mean to leave this change in?

Reply:
No, thanks for catching this. I wonder if maybe the checked-in class shouldn't even contain a main method since it's not included in any of the code snippets.